android/src/main/java/com/swmansion/reanimated/NodesManager.java

```diff
@@ -136,6 +136,7 @@ public NativeUpdateOperation(int viewTag, WritableMap nativeProps) {
     }
   }
   private Queue<NativeUpdateOperation> mOperationsInBatch = new LinkedList<>();
+  private boolean mTryRunBatchUpdatesSynchronously = false;
```

Comment:
Why do we want to opt-out of the synchronous layout when someone uses reanimated 1? Is it possible that we will set this variable to true and some of the REA 1 operations are in batch?

Reply:
this is to keep the behavior of apps that use reanimated 1 unchanged. When no reanimated 2 code is used it will always stay false and will fallback to the old update mechanism. It isn't possible for batch to be split with this as we always process the whole batch at once, just sometimes we will do it synchronously and other times we won't